doc/_static/css/landingpage.css

```diff
@@ -0,0 +1,1526 @@
+html {
```

Comment:
The same holds true for some other images (not all, I think the image containing only the frog logo was part of the api documentation), fonts and misc. files.